ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -326,38 +332,39 @@
     },
     {
       'locale': 'fr',
-      'missingTranslations': true,
       'replaceSync': {
-        'Bladesblood': 'onde de choc',
+        'Bladesblood': 'Onde De Choc',
         'Gunshield': 'bouclier-canon',
         'I shall not yield!': 'Mon pavois est infrangible!',
         'Ignis Est': 'Ignis Est',
         'Magitek Turret II': 'tourelle magitek TM-II',
-        'Phantom Varis': 'double de Varis',
+        'Phantom Varis': 'Double De Varis',
         'Terminus Est': 'Terminus Est',
         'Varis Yae Galvus': 'Varis yae Galvus',
         'Ventus Est': 'Ventus Est',
       },
       'replaceText': {
-        'Aetherochemical Grenado': 'Grenade magitek',
-        'Alea Iacta Est': 'Alea Jacta Est',
-        'Altius': 'Altius',
-        'Citius': 'Citius',
-        'Electrified Gunshield': 'Bouclier-canon : Choc magitek',
-        'Festina Lente': 'Festina Lente',
+        '--clones appear?--': '--Apparition des clones ?--',
+        'Aetherochemical Grenado?': 'Grenade Magitek ?',
+        'Alea Iacta Est?': 'Alea Jacta Est ?',
+        'Altius?': 'Altius ?',
```

Comment:
I think it would be better to have one replaceText that is `'\\?': ' ?',` and translate the other lines individually?

Reply:
ok, ty.